moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -36,62 +36,67 @@
 
 #include <moveit/trajectory_execution_manager/trajectory_execution_manager.h>
 #include <moveit/robot_state/robot_state.h>
-#include <moveit_ros_planning/TrajectoryExecutionDynamicReconfigureConfig.h>
-#include <dynamic_reconfigure/server.h>
+// #include <moveit_ros_planning/TrajectoryExecutionDynamicReconfigureConfig.h>
+// #include <dynamic_reconfigure/server.h>
 #include <tf2_eigen/tf2_eigen.h>
 
 namespace trajectory_execution_manager
 {
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.trajectory_execution_manager");
+
 const std::string TrajectoryExecutionManager::EXECUTION_EVENT_TOPIC = "trajectory_execution_event";
 
-static const ros::Duration DEFAULT_CONTROLLER_INFORMATION_VALIDITY_AGE(1.0);
+static const rclcpp::Duration DEFAULT_CONTROLLER_INFORMATION_VALIDITY_AGE(1.0);
 static const double DEFAULT_CONTROLLER_GOAL_DURATION_MARGIN = 0.5;  // allow 0.5s more than the expected execution time
                                                                     // before triggering a trajectory cancel (applied
                                                                     // after scaling)
 static const double DEFAULT_CONTROLLER_GOAL_DURATION_SCALING =
     1.1;  // allow the execution of a trajectory to take more time than expected (scaled by a value > 1)
-
-using namespace moveit_ros_planning;
+// Generated by the dynamic_reconfigure
```

Comment:
What's this from?

Reply:
I think this namespace is only used for the favor of dynamic_reconfigure.